.gitignore

```diff
@@ -12,3 +12,4 @@ project.lock.json
 
 .DS_Store
 .idea/
+.vscode/*
```

Comment:
The VSCode configuration is deliberately committed to the repo, so let's not do this.

Reply:
Ah. I missed that there was already a .vscode dir because when I opened vs code, it created two more files and I thought these were the only newly generated:

```
	.vscode/launch.json
	.vscode/tasks.json
```

I can just manually ignore those.